IFC4x3/Properties/f/ForeignObjectDetectionSensorInterface_2Jp3IQs6DDhg_MdI05mxpK/Documentation.md

```diff
@@ -0,0 +1 @@
+Indicate the type of interface of foreign object dual power network sensor.
```

Comment:
```suggestion
Indicates the type of foreign object dual power network sensor interface.
```

Reply:
This property is generalized as SerialInterfaceType: https://github.com/bSI-InfraRoom/IFC-Specification/pull/165/commits/27b9265b83f4382712448ac447c96641cdcb43d6